demo/body3d_two_stage_img_demo.py

```diff
@@ -264,7 +264,7 @@ def main():
         vis_3d_pose_result(
             pose_lift_model,
             result=pose_lift_results_vis,
-            img=pose_lift_results[0]['image_name'],
+            img=image_name,
             out_file=out_file)
```

Comment:
What's the purpose of this modification? If it's necessary, please remove the following line.
https://github.com/open-mmlab/mmpose/blob/dc5a4c26578a1edb16c8487971656682c66363fd/demo/body3d_two_stage_img_demo.py#L222

Reply:
Because I add a `_gather_pose_lifter_inputs` function to normalize 'keypoints', where only 'keypoints' and 'track_id' are kept . In this way, the only useful information in `pose_lift_results` is 'keypoints_3d'. Other information such as 'image_name' and unnormalized 'keypoints' can be obtained from `pose_det_results`.